src/components/ItemTable/ItemTable.tsx

```diff
@@ -27,26 +22,25 @@ const ItemTable = ({ itemTableData }: ItemTableProps) => {
           inSearchResult ? " " + styles.inSearchResult : ""
         }`}
         columnHeaders={tableHeadings}
-        // TODO: Review these values with the design team
         columnStyles={
           inSearchResult
             ? [
-                { width: "33.3%", minWidth: 150, maxWidth: 250 },
-                { width: "33.3%", minWidth: 150, maxWidth: 250 },
-                { width: "33.3%", minWidth: 150, maxWidth: 250 },
+                { width: 272, minWidth: 85 },
+                { width: 272, minWidth: 85 },
+                { width: 272, minWidth: 85 },
               ]
             : [
-                { minWidth: 350, maxwidth: 350 },
-                { minwidth: 150, maxWidth: 200 },
-                { minwidth: 150, maxWidth: 150 },
-                { minwidth: 150, maxWidth: 150 },
-                { minwidth: 200, maxWidth: 250 },
-                { minwidth: 150, maxWidth: 200 },
+                { width: "auto", minWidth: 250 },
+                { width: "14%", minWidth: 100 },
+                { width: "14%", minWidth: 100 },
+                { width: "14%", minWidth: 100 },
+                { width: "14%", minWidth: 100 },
+                { width: "14%", minWidth: 100 },
               ]
         }
         tableData={tableData}
         showRowDividers={!inSearchResult}
-        isScrollable={!isLargerThanMobile}
+        isScrollable={true}
```

Comment:
You don't need to explicitly set this to `true`.  It can just be `isScrollable`.